### ISSUE TITLE
Loss equal Nan when finetune the model

### ISSUE THREAD

**Issue by QianshengGu** [hellochick/ICNet-tensorflow#95]
Hi, when I try to finetune the model after 250000 iterations, but I got an nan loss without any ideas, can you help me to solve this problem? Best regards!
![image](https://user-images.githubusercontent.com/33200440/50222307-d69ade80-03d2-11e9-9a1b-34f66f23cf5b.png)
